WordPress/twentytwentyfive
# removing query no results patterns

**Description**

There are two "no results found" patterns, "hidden-no-results-centered.php" and "hidden-no-results.php", which are used only once each. Request is to replace the patterns where used with the translatable text instead to increase performance.
#319

**Testing Instructions**

1. Activate the 2025 theme.
2: Two patterns are updated, 
     "Photo blog posts" where the "No posts were found." text should be centered
     "List of posts, 1 column" where the "No posts were found." text should be left-aligned
3: For both of the above, ensure the "No posts were found." appears when appropriate, and does not show when posts are found.
